cloudinit/distros/freebsd.py

```diff
@@ -121,13 +121,14 @@ def add_user(self, name, **kwargs):
             pw_useradd_cmd.append("-d/nonexistent")
             log_pw_useradd_cmd.append("-d/nonexistent")
         else:
-            pw_useradd_cmd.append(
-                "-d{home_dir}/{name}".format(home_dir=self.home_dir, name=name)
+            homedir = "{home_dir}/{name}".format(
+                home_dir=self.home_dir, name=name
             )
+            if "homedir" in kwargs:
+                homedir = kwargs["homedir"]
+            pw_useradd_cmd.append("-d{homedir}".format(homedir=homedir))
```

Comment:
Nit: This is a little verbose, I think the lines above could be simplified to the following:
```
homedir = kwargs.get("homedir", f"{self.home_dir}/{name}")
pw_useradd_cmd.append("-d" + homedir)
```

Reply:
done!